package.json

```diff
@@ -11,6 +11,7 @@
   "author": "Paul",
   "license": "ISC",
   "devDependencies": {
+    "eslint": "^8.57.0",
```

Comment:
This is a test. Do not like this dependency!

Reply:
ok I'll get on that.

Reply:
Ok Awesome!